### PR TITLE
16466 - QA/ UX tweaks for outputs

### PR DESCRIPTION
**Pull request by seeker25**
*Issue #:* /bcgov/entity#16466

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Review by argush3 — APPROVED** [review]

File: legal-api/src/legal_api/core/meta/filing.py [hunk 1]
```diff
@@ -502,14 +502,17 @@ def alter_outputs_alteration(filing, outputs):
     def alter_outputs_correction(filing, business, outputs):
         """Handle output file list modification for corrections."""
         if filing.filing_type == 'correction':
+            if filing.meta_data.get('correction', {}).get('toLegalName'):
+                outputs.add('certificateOfNameChange')
```
Comment (seeker25) [inline legal-api/src/legal_api/core/meta/filing.py:506]: Missing in outputs for correction

File: legal-api/src/legal_api/core/meta/filing.py [hunk 2]
```diff
@@ -502,14 +502,17 @@ def alter_outputs_alteration(filing, outputs):
...
             corrected_filing_id = filing.filing_json.get('correction', {}).get('correctedFilingId')
             original_filing = FilingStorage.find_by_id(corrected_filing_id)
             if is_special_resolution_correction(
                     filing.filing_json['filing'], business, original_filing
                     ):
                 if filing.filing_json['filing']['correction'].get('rulesFileKey'):
                     outputs.add('certifiedRules')
-                outputs.add('specialResolution')
+                if filing.meta_data.get('correction', {}).get('hasResolutionChanges'):
+                    outputs.add('specialResolution')
```
Comment (seeker25) [inline legal-api/src/legal_api/core/meta/filing.py:515]: Hiding this if there are no resolution changes. Only documents that show change hopefully.

File: queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/correction.py [hunk 3]
```diff
@@ -109,6 +109,9 @@ def correct_business_data(business: Business,  # pylint: disable=too-many-locals
         resolution = dpath.util.get(correction_filing, '/correction/resolution')
         filings.update_filing_json(correction_filing_rec, resolution)
         resolutions.update_resolution(business, resolution)
+        if resolution:
+            filing_meta.correction = {**filing_meta.correction,
```
Comment (seeker25) [inline queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/correction.py:113]: Hiding this if there are no resolution changes. Only documents that show change hopefully.

**Pull request merged**